omap/modules/base_settings.py

```diff
@@ -52,6 +52,7 @@
     "django.contrib.messages.middleware.MessageMiddleware",
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
 ]
+ROOT_URLCONF = "omap.modules.module_urls"
```

Comment:
hm... why is that in here? or rather... why is this a change here? Wasnt that in here already?